3rdparty/find_dependencies.cmake

```diff
@@ -901,14 +922,22 @@ if(BUILD_GUI)
     set(FILAMENT_MATC "${FILAMENT_ROOT}/bin/matc")
     target_link_libraries(3rdparty_filament INTERFACE Threads::Threads ${CMAKE_DL_LIBS})
     if(UNIX AND NOT APPLE)
-        find_library(CPPABI_LIBRARY c++abi PATH_SUFFIXES
-            llvm-11/lib llvm-10/lib llvm-9/lib llvm-8/lib llvm-7/lib
-            REQUIRED)
-        get_filename_component(CPP_LIBDIR ${CPPABI_LIBRARY} DIRECTORY)
-        find_library(CPP_LIBRARY c++ PATHS ${CPP_LIBDIR} REQUIRED)
+        # Find CLANG_LIBDIR if it is not defined. Mutiple paths will be searched.
+        if (NOT CLANG_LIBDIR)
+            find_library(CPPABI_LIBRARY c++abi PATH_SUFFIXES
+                         llvm-11/lib llvm-10/lib llvm-9/lib llvm-8/lib llvm-7/lib
```

Comment:
Is this needed any more after setting `CLANG_LIBDIR` explicitly?

Reply:
Yes, this is needed. `CLANG_LIBDIR` is only explicitly set when `BUILD_FILAMENT_FROM_SOURCE=ON`. Otherwise, we need to find `CLANG_LIBDIR` again.